### DISCUSSION TITLE
Sayı grupları sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#33] [Zeka Sorusu] [answered]
0 dan 14 e kadar olan sayılar şu şekilde gruplandırılmış:
Grup A: 0, 3, 6, 8, 9
Grup B: 1, 4, 7, 11, 14
Grup C: 2, 5, 10, 12, 13
Buna göre 15, 16 ve 17 hangi gruplara dahil edilmelidir?

**Comment by Berkek007** [comment, marked as answer]
Cevap: Grup A da sayılar yuvarlatılmış çizgilerle oluşturulmuş, Grup B de sayılar düz olan çizgilerden oluşturulmuş, Grup C de ise sayılar hem yuvarlatılmış hem de düz çizgilerden oluşturulmuştur. Bu sebeple 15 ve 16 Grup C ye, 17 ise Grup B ye dahil edilmelidir.
